docker-compose.yml

```diff
@@ -43,6 +43,13 @@ services:
       - ./server:/apps/warehouse/server
       - ./package.json:/apps/warehouse/package.json
       - ./npm-shrinkwrap.json:/apps/warehouse/npm-shrinkwrap.json
+    depends_on:
+      - db
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://web:3000/api/StoreModels"]
```

Comment:
also @pulkitsinghal you can provide `--start-period=DURATION ` for healthchecks to start.
This is in case you want boot scripts to finish before healthcheck starts
more at
https://docs.docker.com/engine/reference/builder/#healthcheck